RadixWallet/RadixWalletDebug-Dev.entitlements

```diff
@@ -3,9 +3,7 @@
 <plist version="1.0">
 <dict>
 	<key>com.apple.developer.icloud-container-identifiers</key>
-	<array>
-		<string>iCloud.com.radixpublishing.radixwallet.ios.dev.cloudBackup</string>
-	</array>
+	<array/>
```

Comment:
What is the inferred iCloud container identifier? given that this array is empty

Reply:
The default name, which is `iCloud.[bundle ID]`

Reply:
then why Pre-Alpha and Alpha have the iCloud container explicitly specified? why the difference?

Reply:
it makes no difference when running it, except that in some cases I had to add it explicitly for xcode  to pick up the changes

Reply:
I would opt to be consistent, either the container are explicitly specified or not, I more for explicitly specifying these.

Reply:
sure, already did that